### PR TITLE
5834 x1 for mainnet - Main

### PR DESCRIPTION
**Pull request by prathmeshkhandelwal1**
## Description

Feat: Adding x Layer

## Type of change

- [ ] Docs change / dependency upgrade
- [ ] Configuration / tooling changes
- [ ] Refactoring
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires changes in customer code

## Related Issue(s)

Fixes

### CODE REVIEW

**Review by preethamr — COMMENTED** [review]
Do we need the graph client built to the latest state of the branch ?
LGTM otherwise

**Comment by prathmeshkhandelwal1** [comment]
> Do we need the graph client built to the latest state of the branch ? LGTM otherwise

I dont think so, as the latest state of `main` branch and after adding x layer I rebuild it so it should include both

File: packages/agents/sdk/src/config.ts [hunk 1]
```diff
@@ -214,9 +215,10 @@ export const XERC20REGISTRY_DOMAIN_ADDRESS: Record<string, string> = {
   "1650553709": "0xcc1ce162fb1d70a377d1542873b1db57f5ff14d1", // base
   "1836016741": "0x689cCB572262e0ca66146FE5Ed5D998543d3AA3B", // mode
   "1935897199": "0x397aEEEDd44f40326f9eB583a1DFB8A7A673C40B", // scroll
+  "2020368761": "", // xlayer
```
Comment (just-a-node) [inline packages/agents/sdk/src/config.ts:218]: Registry + adapter to be deployed in separate PR?

**Review by just-a-node — APPROVED** [review]

File: packages/agents/sdk/src/config.ts (hunk 1 above)
Comment (prathmeshkhandelwal1) [reply packages/agents/sdk/src/config.ts:218]: yes, Will add it separately in sdk
